bender.js

```diff
@@ -24,41 +24,42 @@
  */
 
 var config = {
-    applications: {
-        ckeditor: {
-            path: '.',
-            files: [
-                'ckeditor.js'
-            ]
-        }
-    },
+	applications: {
+		ckeditor: {
+			path: '.',
+			files: [
+				'ckeditor.js',
+				'tests/polyfills/bind.js'
```

Comment:
This bind polyfill is now useless since I actually jogged my memory.  I'm now +0 on keeping this around.